src/metabase/search/appdb/index.clj

```diff
@@ -256,18 +258,16 @@
         (swap! *indexes* assoc :pending nil))))
 
   (let [entries          (map document->entry documents)
-        ;; Optimization idea: if the updates are coming from the re-indexing worker, skip updating the active table.
-        ;;                    this should give a close to 2x speed-up as insertion is the bottleneck, and most of the
-        ;;                    updates will be no-ops in any case.
-        active-updated?  (safe-batch-upsert! (active-table) entries)
+        ;; No need to update the active index if we are doing a full index and it will be swapped out soon. Most updates are no-ops anyway.
```

Comment:
Worth mentioning that these inserts are the bottleneck, so we're doing this for a large expected speed-up.

Reply:
Yeah, getting to this part of the change is overall point of this PR and I prioritized it for the hopeful performance boost.

Reply:
Sorry I just meant worth mentioning in this comment

Reply:
💯 Just agreeing with you